ui/cryptomaterial/dropdown.go

```diff
@@ -316,6 +316,7 @@ func (d *DropDown) collapsedAndExpandedLayout(gtx C) D {
 // expandedLayout computes dropdown layout when dropdown is opened.
 func (d *DropDown) expandedLayout(gtx C) D {
 	m := op.Record(gtx.Ops)
+	gtx.Constraints.Max.Y = inf
```

Comment:
what is inf? @bugjt

Reply:
This makes the dropdown menu display on the window size, regardless of the size of its parent that contains it.

Reply:
It fixes this bug: "1. The server and market selection on the dex market page no longer stack on top of every element on the page when the dropdown is expanded"